.pipelines/stages/jobs/steps/capi-linux-step.yml

```diff
@@ -132,12 +132,6 @@ steps:
     displayName: 'Build Python $(PyNoDotVer)'
     workingDirectory: '$(Build.Repository.LocalPath)'
 
-  - task: BinSkim@4
```

Comment:
Just to confirm that we removed BinSkim becaused it is covered under sdl

Reply:
Yes